bitnami/dataplatform-bp1/Chart.yaml

```
@@ -1,7 +1,7 @@
 annotations:
   category: Infrastructure
 apiVersion: v2
-appVersion: 1.0.0
+appVersion: 2.0.0
```

As there is no standalone application behind this chart, let's use the same app version as the one used as the chart version (see below comment)
```suggestion
appVersion: 0.2.0
```

---

In the first release, the appVersion was 1.0.0 and the chart version was 0.1.0. 
According to us, this is a major release for the data platform chart, hence we went with 2.0.0 and 1.0.0.

---

The `appVersion` is not used by Helm to generate the package tarball, it is considered as metadata. This version points to the version of the bundled application in the Helm Chart. For example, in the PostgreSQL Helm chart, this field is `appVersion: 11.11.0` which is the version of the PostgreSQL container used by the Helm Chart.

Then we have the `version` field which is the version of the chart itself (nothing related to the application inside the chart). This version should be modified every time the chart is modified (changes in the values.yaml, notes.txt, k8s manifests, etc). This version follows [semver](https://semver.org/spec/v2.0.0.html). According to semver, the patch version is bumped when the changes are fixing bugs, the minor version when some feature is added and the major version when the backward compatibility is not guaranteed. +info: https://helm.sh/docs/topics/charts/#charts-and-versioning

---------------------------------------

The above explanation is how the version works in Helm as a general rule. In the case of this dataplatform chart, there are some exceptions because there is not any main component bundled in the chart to be used to set the `appVersion` according to it, we can say this chart is not bundling an application so we can consider the chart itself as the product/application, that is why we are suggesting to use the same version in the `appVersion` and `version` field.

In the rest of the charts, the `appVersion` is set according to the upstream maintainers, I mean, if PostgreSQL releases a new version such as 12.4.5 and we update the chart to use it, the `appVersion` is going to reflect this change.

The `version` is not metadata, this field is used to package the chart and it is important to follow the semver directives

---

@miguelaeh maybe you have more context about this topic and I'm missing something, what do you think the relationship between `appVersion` and `version` should be?

---

I have made the changes to the versions, please have a look.

---

Thanks for the change, now the `version` was bumped in a minor what is fine, but the `appVersion` field is still pointing to `1.0.0` that was the original value (probably wrong in the first iteration of the Helm chart).

The topic is: `version` is going to be updated at the moment in which there is any change in the chart (even a minor change in a comment), but what about `appVersion`? The purpose of this field is to serve as metadata to show what is the version used by the bundled application, but in this case, there is not a single application. What is your plan to bump `appVersion`? Where should we look to detect that there is a new version? What we usually do in Bitnami in this kind of chart where there is not a single app is set the same value used in `version` and bump both values at the same time.

---

Hi @carrodher Considering we had 1.0.0 as the appversion in the first one, we can let that be. 
The idea is to have different sizes of the data platform clusters.
Example: Current appversion 1.0.0 is related to 'Small' data platform cluster. Next major version i.e. appversion 2.0.0 will be when we add the 'Medium'/'Large' data platform cluster. 
Hope this clarifies.

---

Ok, as it is something that you are going to handle due to the lack of application when the chart evolves